webpreview/previews.py

```diff
@@ -148,10 +148,10 @@ def _set_properties(self):
                 new_property = re.sub('(?!^)([A-Z]+)', r'_\1',property).lower()
             else:
                 new_property = property
-
-            if property_meta and property_meta['content'] != "":
+            # to fix keyError https://www.crummy.com/software/BeautifulSoup/bs4/doc/#miscellaneous
+            if property_meta and property_meta.get('content') != "":
                 # dynamically attach property to instance
-                self.__dict__[new_property] = property_meta['content']
+                self.__dict__[new_property] = property_meta.get('content')
             else:
                 self.__dict__[new_property] = None
```

Comment:
I feel like this entire section can be replaced with
```diff
-            # to fix keyError https://www.crummy.com/software/BeautifulSoup/bs4/doc/#miscellaneous
-            if property_meta and property_meta['content'] != "":	            if property_meta and property_meta.get('content') != "":
-                # dynamically attach property to instance	                # dynamically attach property to instance
-                self.__dict__[new_property] = property_meta['content']	                self.__dict__[new_property] = property_meta.get('content')
-            else:	            else:
-                self.__dict__[new_property] = None	                self.__dict__[new_property] = None
+           if property_meta:
+               self.__dict__[new_property] = property_meta.get('content')
```

Has the same effective end result, but is cleaner

Reply:
scratch that, the diff is alright